go.mod

```diff
@@ -3,46 +3,36 @@ module github.com/nyaruka/courier
 require (
 	github.com/antchfx/xmlquery v0.0.0-20181223105952-355641961c92
 	github.com/antchfx/xpath v0.0.0-20181208024549-4bbdf6db12aa // indirect
-	github.com/aws/aws-lambda-go v1.8.1 // indirect
-	github.com/aws/aws-sdk-go v1.13.3
+	github.com/aws/aws-sdk-go v1.34.17
 	github.com/buger/jsonparser v0.0.0-20180318095312-2cac668e8456
 	github.com/certifi/gocertifi v0.0.0-20180118203423-deb3ae2ef261 // indirect
 	github.com/dghubble/oauth1 v0.4.0
 	github.com/evalphobia/logrus_sentry v0.4.6
 	github.com/garyburd/redigo v1.5.0
 	github.com/getsentry/raven-go v0.0.0-20180517221441-ed7bcb39ff10 // indirect
-	github.com/go-chi/chi v3.3.3+incompatible
+	github.com/go-chi/chi v4.1.2+incompatible
```

Comment:
I didn't mean to bump our version of chi but gocommon ended up with this version.. and it's working fine with mailroom